SECURITY.md

```diff
@@ -0,0 +1,55 @@
+# Securing the dashboard
+
+The following page describes some ways to secure the dashboard.
+
+## Basic Authentication on Istio with Envoy Filter
+When you are using Istio there is a way of introducing an envoy filter which contains
+a way of introducing basic authentication for a certain domain and path.
+
+This assumes you have the dashboard exposed through the istio ingress on "example.com/build".
+
+```
+apiVersion: networking.istio.io/v1alpha3
+kind: EnvoyFilter
+metadata:
+  name: basic-auth-tekton-dashboard
+  namespace: istio-system
+spec:
+  workloadLabels:
+    app: istio-ingressgateway
+  filters:
+  - listenerMatch:
+      listenerType: GATEWAY
+    filterType: HTTP
+    filterName: envoy.lua
+    filterConfig:
+      inlineCode: |
+        local domain = "example.com"
+        local path = "/build/"
+
+        -- base64 encoded: <username>:<password>
+        -- example: ok:test2
+        local base64UsernamePassword = "Basic b2s6dGVzdDI=" 
```

Comment:
So this is the value you'd change to set it to whatever you fancy right? Let's call that out with a helpful comment